### PR TITLE
rename Makefile.dune to Makefile

### PR DESCRIPTION
**Pull request by Alizter**

### CODE REVIEW

**Comment by SkySkimmer** [comment]
>I think I've messed up the test-suite/coq-makefile/timing/precomputed-time-tests/007-no-output-sync/time-of-build.log.in file. What is the correct way to update it?

CI seems ok with it.
cc @JasonGross

**Comment by Alizter** [comment]
Or nevermind I guess seems to work.

File: dev/doc/README.md [hunk 1]
```diff
@@ -44,10 +44,9 @@ the one used in the Coq repository), e.g.:
 
 ## Building `coqtop` / `coqc` binaries
 
-We recommend that you use the targets in `Makefile.dune`.  See
-[`build-system.dune.md`](build-system.dune.md) to learn more about
-them. `Makefile` is just an alias for `Makefile.dune` so you can just
-call `make` directly.
+We recommend that you use the targets in the `Makefile`. Calling `make` will
+list out available targets. See [`build-system.dune.md`](build-system.dune.md)
```
Comment (jfehrle) [inline dev/doc/README.md:48]:
```suggestion
show the available targets. See [`build-system.dune.md`](build-system.dune.md)
```

**Review by jfehrle — APPROVED** [review]
one suggestion

**Comment by silene** [comment]
The fact that the testsuite times out after 3 hours is a bit worrying. It usually takes about 45 minutes: https://gitlab.com/coq/coq/-/jobs/2998074863 https://gitlab.com/coq/coq/-/jobs/2740903629 https://gitlab.com/coq/coq/-/jobs/3033316827 https://gitlab.com/coq/coq/-/jobs/3019175090

**Comment by SkySkimmer** [comment]
It happens sometimes, we have no idea why.

**Comment by Alizter** [comment]
The test-suite issue should be unrelated. I think it was hypothesized on Zulip being due to the runner being slow on memory and spending all its time swapping.

**Comment by jfehrle** [comment]
FWIW, it would be good to examine how much memory the test suite has when it times out.  This info is already in the log file.  Maybe it could abort earlier if there's not enough memory.

**Comment by Alizter** [comment]
Two timeouts.... trying again.

**Comment by jfehrle** [comment]
Did you check how much memory the machine has for the bad runs?

**Comment by silene** [comment]
> Did you check how much memory the machine has for the bad runs?

From the inside of the virtual environment, good and bad machines are indistinguishable, they report the exact same memory limits, that is, they have about 3GB of free memory. But who knows how many of it is a lie.

**Comment by SkySkimmer** [comment]
>Two timeouts.... trying again.

These timeouts have nothing to do with this PR, why retry?

**Comment by Zimmi48** [comment]
If we cannot distinguish the runners with low memory, could we at least prevent swapping / detect swapping and fail early?

**Comment by Alizter** [comment]
Can I get an assignee?

**Comment by jfehrle** [comment]
> If we cannot distinguish the runners with low memory, could we at least prevent swapping / detect swapping and fail early?

The script could do a timing check, e.g. how long it takes to run some subset of the test suite or the time required for each test.  To figure it out, one would probably want to hack the test runner script to print the time used per test and the elapsed time.

**Review by SkySkimmer — APPROVED** [review]

**Comment by SkySkimmer** [comment]
@coqbot merge now

**Comment by ejgallego** [comment]
cc #6137

**Pull request merged**